export function handleErrorMessage() {
    alert("Something went wrong, please try again later");
}
